src/10_Introduction/Internationalization.html

```diff
@@ -0,0 +1,104 @@
+</head>
+<body>
+<!--
+
+## Introduction
+
+How to support hreflang and internationalization with AMP.
+
+## About Internationalization
+
+Internationalization and localization of content is an important feature that
+websites often embrace to provide maximum reach for their content or services.
+
+For static websites, a common site structure technique for addressing this is
+called 'hreflang'. Hreflang is a part of the meta data of a website and is used
+to annotate a page such that crawlers can discover alternative translations of
+a particular article. Search engines such as Google recognise websites that use hreflang.
+
+You can [read more about hreflang here.](https://support.google.com/webmasters/answer/189077?hl=en)
+
+AMP documents can also use the hreflang notation to point crawlers towards
+other versions of themselves in other languages.
+
+However, it's important to correctly setup the hreflang relationships between
+AMP documents correctly. Especially when one considers the existing meta data
+relationships between Canonical documents and AMP documents. Below are two examples
+of how websites might structure their hreflang relationships for Canonical and AMP
+documents across an article.
+
+## Example 1: Canonical + AMP
+
+The most common site structure pattern is to support translations of an article
+with a canonical document and a corresponding AMP document.
+
+The following diagram illustrates this structure at a high level:
+
+[<amp-img layout="responsive" src="/internationalization/amp_hreflang_example_1.png" width="960" height="540"></amp-img>](/internationalization/amp_hreflang_example_1.png)
+
+In this diagram, there is an English canonical desktop document and
+a corresponding AMP document. As per the official AMP Project documentation for
+[discovery and distribution](https://www.ampproject.org/docs/get_started/create/prepare_for_discovery)
+the English AMP document must include a link tag to the canonical document. Similarly, the
+canonical document includes a corresponding link tag to the AMP document.
+
+Additionally, this diagram includes French translations of both the canonical document
+and the AMP document. Therefore the documents must include link tags with hreflang
+attributes to connect the various alternative language versions of the canonical and
+AMP documents. The following rules apply:
+* Every canonical document must have a link tag with rel=amphtml pointing to
```

Comment:
Can you please surround code snippets such `rel=amphtml` and `rel=canonical` with ` `` `.